server/order/order.go

```diff
@@ -77,6 +80,10 @@ type Order interface {
 
 	// Remaining computes the unfilled amount of the order.
 	Remaining() uint64
+
+	// SwapAddress returns the order's payment address. Will be empty string for
+	// CancelOrder.
+	SwapAddress() string
```

Comment:
I'm not crazy about requiring the `CancelOrder` type to implement this method, and I'm open to other solutions.